### PR TITLE
Noosefix

### PR DESCRIPTION
**Pull request by Crystalwarrior**
fixes nooses not detecting chairs

### CODE REVIEW

File: code/modules/power/cable.dm [hunk 1]
```diff
@@ -512,13 +512,13 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list ( \
...
 	if(R.title == "noose")
-		if(!(locate(/obj/structure/stool) in usr.loc) && !(locate(/obj/structure/table) in usr.loc) && !(locate(/obj/structure/toilet) in usr.loc))
+		if(!(locate(/obj/structure/stool) in usr.loc) && !(locate(/obj/structure/bed) in usr.loc) && !(locate(/obj/structure/table) in usr.loc) && !(locate(/obj/structure/toilet) in usr.loc))
 			usr << "<span class='warning'>You have to be standing on top of a chair/table/toilet to make a noose!</span>"
 			return 0
 	return ..()
 
 /obj/item/stack/cable_coil/suicide_act(mob/living/user)
-	if((locate(/obj/structure/stool) in user.loc) || (locate(/obj/structure/table) in user.loc) || (locate(/obj/structure/toilet) in user.loc))
+	if((locate(/obj/structure/stool) in user.loc) || !(locate(/obj/structure/bed) in usr.loc) || (locate(/obj/structure/table) in user.loc) || (locate(/obj/structure/toilet) in user.loc))
```
Comment (Carbonhell) [inline code/modules/power/cable.dm]: Why !locate

Comment (Carbonhell) [inline code/modules/power/cable.dm]: Answer nerd @crystalwarrior160

**Comment by Crystalwarrior** [comment]
Merge pls.

**Comment by Carbonhell** [comment]
> merge
> didn't even care to answer my line note

File: code/modules/power/cable.dm (hunk 1 above)
Comment (Crystalwarrior) [inline code/modules/power/cable.dm]: oops, comitting a fix

**Comment by Carbonhell** [comment]
nerd. merging

**Pull request merged**